### PR TITLE
use usequery and useMutation hooks

### PR DESCRIPTION
**Pull request by jgfidelis**
#### What is the purpose of this pull request?
use react-apollo@hooks

#### What problem is this solving?
https://minicart--alssports.myvtex.com/

tests are breaking because vtex-test-tools is on react-apollo 2.x yet, need to fix it in there

#### Types of changes
- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Requires change to documentation, which has been updated accordingly.

### CODE REVIEW

File: react/index.js [hunk 1]
```diff
@@ -152,19 +160,22 @@ const MiniCart = ({
...
-  updateOrderForm,
   intl,
-  updateItemsMutation,
-  addToCartMutation,
-  updateLocalItemStatus,
 }) => {
+  const client = useApolloClient()
   useLinkState(client)
 
+  const { loading, data={}} = useQuery(orderFormQuery, {ssr:false})
```
Comment (lucasecdb) [inline react/index.js]: are you running prettier on these files?

File: react/index.js [hunk 2]
```diff
@@ -7,6 +7,7 @@ import {
   pick,
   isNil,
   prop,
+  compose,
```
Comment (lucasecdb) [inline react/index.js]: do you need this?

**Comment by lucasecdb** [comment]
@jgfidelis can you provide a workspace with this changes linked?

File: react/index.js (hunk 2 above)
Comment (jgfidelis) [reply react/index.js]: yes

**Review by lucasecdb — CHANGES_REQUESTED** [review]
please remove the commented code

**Review by lucasecdb — APPROVED** [review]
LGTM

File: react/index.js [hunk 3]
```diff
@@ -337,7 +349,15 @@ const MiniCart = ({
       }
     },
     // eslint-disable-next-line react-hooks/exhaustive-deps
```
Comment (klzns) [inline react/index.js:351]: Why does it have that comment? Do you know? What happens if we remove it?

**Review by klzns — APPROVED** [review]

**Comment by Klynger** [comment]
Going to close this PR due to inactivity

**Pull request closed without merging**